src/node.cpp

```diff
@@ -1628,7 +1633,7 @@ LocalNode::~LocalNode()
     }
 }
 
-void LocalNode::getlocalpath(string* path, bool sdisable) const
+void LocalNode::getlocalpath(string* path, bool sdisable, const std::string* localseparator) const
```

Comment:
we are always using the client's fsaccess's localseparator, no need for this change?

Reply:
We need this for unit testing. Similar to `setfsid`